docs/conf.py

```diff
@@ -84,6 +84,7 @@
     r"http://localhost:\d+/",
     r"http://127\.0\.0\.1:\d+/",
     r"https://.*sentry\.openzaak\.nl.*",
+    r"https://miniwebtool.com",  # cannot be fetched with cURL or non-browser
```

Comment:
Some anything spam thing probably prevents cURL from requesting this site, gets a 403 response

Reply:
The link to miniwebtool should already be removed from the docs, let's not include it in the list of trusted urls

Reply:
Removed it from the rest of the documentation (and some comments too)